tests/bigbang_tests.py

```diff
@@ -122,6 +122,9 @@ def test_labeled_blockmodel():
 def test_empty_list_compute_activity_issue_246():
     test_df_csv_path = os.path.join(CONFIG.test_data_path, 'empty-archive-df.csv')
     df = pd.DataFrame.from_csv(path=test_df_csv_path)
-    activity = archive.Archive(df).get_activity()
 
-    assert True, "activity computation crashed on empty df"
\ No newline at end of file
+    with assert_raises(mailman.MissingDataException):
+        empty_archive = archive.Archive(df)
+        activity = empty_archive.get_activity()
+
+    assert True, "activity computation on empty df without MissingDataException"
```

Comment:
I'm confused why this assertion is still here. Wouldn't you want to make sure that this line was never reached, because of the raised error? wouldn't that mean this should be assert False?

Reply:
Just tried that. An assert False there fails, I guess because you get to the line and it fails. But then...wouldn't you just want this removed?

Reply:
I might have read the documentation wrong (regarding assert True vs. assert False). That line should never be reached if the functionality is behaving correctly, but I wanted to put an assert there so that if that line _is_ reached, the assert is thrown and the test fails.

Reply:
Yep, I was confused. This line wasn't doing anything and if I changed it so that it was doing something (assert False from within the with: block) then it just becomes redundant. So I've removed it. Thanks for catching the mistake!